### PR TITLE
Task8

### PR DESCRIPTION
**Pull request by hitansivekariya33**

### CODE REVIEW

File: task8.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+'''
+
+#sort list without in-built function
+numbers = [2, 5, 6, 1, 3, 8, 9, 10]
+for i in range (len(numbers)):
+    for j in range(len(numbers)):
+        if(numbers[i] < numbers[j]):
+            temp = numbers[i]
+            numbers[i] = numbers[j]
+            numbers[j] = temp
```
Comment (himanshu-suthar-tagline) [inline task8.py:16]: Please tell me which algorithm you have used here for sorting.

Comment (hitansivekariya33) [reply task8.py:16]: Swapping using third variable

Comment (himanshu-suthar-tagline) [reply task8.py:16]: It's a logic and description. I asked sorting algorithm name. Which sorting algorithm you are using here to sort the list?

Comment (hitansivekariya33) [reply task8.py:16]: bubble sort algorithm

**Pull request merged**